### ISSUE TITLE
Quire IIIF QA 

### ISSUE THREAD

**Issue by Erin-Cecele** [thegetty/quire#815]
### Before proceeding, make sure there isn’t an existing issue for this bug.

- [X] I have searched the existing issues and determined this is a new bug.

### Expected Behavior

This story is to capture all issues related to the core functioning of basci IIIF zooming images.

We've pushed some assets and content to the `QA-iiif` branch of the `quire-starter-iiif` repo as well to help with testing.

- [x] print-image files are transformed to jpg but are being called based on their source format
- [x] Only .jpgs are working as zooming images
- [x] .jp2 images that are set to zoom cause a previewing/build error
- [x] Zooming images below a certain threshold size don't display in the lightbox
- [x] IIIF manifests outputting with extra slash
- [ ] High-res images are included in _site even when not needed
- [ ] Add +/- bottons to lightbox / canvas panel to control zooming
- [x] The print-image.jpg files need to be a larger size for print-ready PDF
- [ ] Enable tiling PNGs
- [x] IIIF images are ghosting duplicates with any interaction in the modal
- [x] External IIIF manifests are not rendering
- [ ] External IIIF images should only need `printImage` not also `src`
- [ ] Paths to `printImage` file are incomplete in PDF and EPUB output
- [ ] External `printImage` should work in the table of contents grid
- [ ] Use of info.json as `src` is no longer functional or needed
- [ ] Refactor out use of camelCase for external IIIF in figures.yaml?
- [ ] Use the first `canvasId` by default if one is not supplied along with the `manifestId`?

### Actual Behavior

N/A

### Steps to Reproduce

N/A

### Version Numbers

N/A

**Comment by Erin-Cecele** [comment]
Updates: 

- Enable tiling PNGs - this is currently blocked
- IIIF images are ghosting duplicates with any interaction in the modal - this is currently blocked; see #809 for more information

**Comment by cbutcosk** [comment]
@Erin-Cecele I'm not sure if this is already included in your list above, but fullscreen buttons do not appear to work on any browser at the moment. Or maybe they're being factored out?

At any rate if you start any new quire pub and click a figure, the "fullscreen" button in the upper left does nothing and produces an error about `classList` in the console. Happens on Safari 16.6 and Chrome 116. Tested today with quire-1.0-rc.14

**Comment by Erin-Cecele** [comment]
@cbutcosk, thank you for reporting this. I just fired up a new project and noticed that neither the images in the "Introduction" or Figure 1 and Figure 2 in "I. American Photographs"  go fullscreen when you click the button in the upper left. 

However, in "Cat. 1" Figure 1.1 and Figure 1.2 do open in fullscreen when the button is clicked.  Also, both main catalogue entry images also open just fine. Can you confirm if the entry page images are going fullscreen for you? I wonder if the issue is somehow tied to `layout` type?

**Comment by Erin-Cecele** [comment]
Oh and here's my work environment information: 

[Project]
 quire-cli 1.0.0-rc.10
 quire-11ty 1.0.0-rc.14
 starter https://github.com/thegetty/quire-starter-default@2.6.0
[System]
 quire-cli 1.0.0-rc.10
 node v18.16.0
 npm 9.5.1
 os Darwin 21.6.0

And I tested with Chrome  114 and Firefox 117 - same results.

**Comment by cbutcosk** [comment]
@Erin-Cecele Yes confirming the entry page seems to work where the essay full screen buttons do not. I suspect your developers also need to add a Shadow DOM polyfill for Safari < 16.5 (mobile and desktop), which in this issue breaks in a slightly different way from the Chromium-based browsers.

**Comment by Erin-Cecele** [comment]
Hi @cbutcosk, please note I have opened a separate issue to address the fullscreen bug you spotted. See: #849

**Comment by Erin-Cecele** [comment]
Status update as of 10/11

- [ ] High-res images are included in _site even when not needed - SELECTED FOR DEVELOPMENT
- [ ] Add +/- bottons to lightbox / canvas panel to control zooming - BACKLOG
- [ ] Enable tiling PNGs - BLOCKED
- [ ] IIIF images are ghosting duplicates with any interaction in the modal - BLOCKED
- [ ] External IIIF images should only need `printImage` not also `src` - BACKLOG
- [ ] Paths to `printImage` file are incomplete in PDF and EPUB output - BACKLOG
- [ ] External `printImage` should work in the table of contents grid - BACKLOG
- [ ] Use of info.json as `src` is no longer functional or needed - BACKLOG
- [ ] Refactor out use of camelCase for external IIIF in figures.yaml? - BACKLOG
- [ ] Use the first `canvasId` by default if one is not supplied along with the `manifestId`? - BACKLOG

**Comment by Erin-Cecele** [comment]
Only update as of 11/30 is "High-res images are included in _site even when not needed" is backlogged and not selected for development.
